aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,79 @@
+  :local:
+  :depth: 1
+
+What are Views?
+---------------
+
+In the Model-View-Controller (MVC) pattern, the *view* encapsulates the presentation details of the user's interaction with the app. Views are templates that include HTML with embedded code that generates content to send to the client. The code uses :doc:`razor syntax <razor>`, which have deep understanding of HTML that allows it to interact with the HTML in the document with minimal code or ceremony.
+
+ASP.NET Core MVC views are *.cshtml* files stored by default in a *Views* folder within the application. Typically, each controller will have its own folder, in which are views for each of the controller actions that returns a view. 
+
+.. image:: overview/_static/views_solution_explorer.png
+
+In addition to action-specific views, :doc:`partial views <partial>`, :doc:`layouts, and other special view files <layout>` can be used to help reduce repetition and allow for reuse within the app's views.
+
+Creating a View
+^^^^^^^^^^^^^^^
+
+To create a view, first navigate to the */Views* folder of your app. If the view will be used by multiple actions, you may want to create it in the *Shared* folder. Otherwise, create or open a folder for the controller that houses the action that will return the view. Name the file the same as the action, and add a *.cshtml* file extension. For example, to create a view for the *List* action on the *Products* controller, you would create the *List.cshtml* file in the */Views/Products* folder.
+
+How do Controllers Specify Views?
+---------------------------------
+
+Views are typically returned from actions as a :dn:cls:`~Microsoft.AspNetCore.Mvc.ViewResult`. Your action method can create and return a ``ViewResult`` directly, but more commonly if your controller inherits from :dn:cls:`~Microsoft.AspNetCore.Mvc.Controller`, you'll simply use the ``View`` helper method, as this example demonstrates:
+
+.. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Controllers/HomeController.cs
+  :language: c#
+  :lines: 9-14
+  :emphasize-lines: 5
+  :dedent: 4
+  
+The ``View`` helper method has several overloads to make returning views easier for app developers. You can optionally specify a specific view to return, as well as a model to pass to the view. With no parameters, the default convention of using a view name that corresponds to the action name is used.
+
+View Discovery
+^^^^^^^^^^^^^^
+
+When an action returns a view, a process called *view discovery* takes place. This process determines which view file will be used. There are several scenarios, based on how the view was specified from the action method.
+
+``return View();``
+    The most common approach. In this case the framework will use a view with the same name as the action (with a *.cshtml* extension) located in a folder named the same as the controller. Otherwise, it will look in the *Shared* folder for a view with a name corresponding to the action name.
```

Comment:
The runtime will search in the following order:
1. Views/<controller name>/<action name>.cshtml
2. Views/Shared/<action name>.cshtml